pkg/sql/catalog/lease/lease_internal_test.go

```diff
@@ -1203,3 +1204,259 @@ func TestReadOlderVersionForTimestamp(t *testing.T) {
 		})
 	}
 }
+
+// TestTableDescriptorByteSizeOrder inserts different amount of data in each
+// Table and guarantees the relative order of the TableDescriptor sizes are
+// correct.
+func TestTableDescriptorByteSizeOrder(t *testing.T) {
```

Comment:
refactor all of these tests into just one test so you can mix and match all the types of descriptors. For the names, you can [3]string or something and make the join in the namespace table accordingly or something.